app/store/services/login.js

```diff
@@ -1,6 +1,7 @@
 // normalizr
 import { schema, normalize } from "normalizr";
 
+// config
```

Comment:
you should be kidding me! ;)
We have to argue about comment on source files.
To be consistent, if Ido remove this line, I have to clean all project source files.
This could be done later.